omod-1.10/src/main/java/org/openmrs/module/attachments/rest/AttachmentResource1_10.java

```diff
@@ -28,12 +30,14 @@
 import org.springframework.web.multipart.MultipartFile;
 
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 
 import static org.openmrs.module.attachments.AttachmentsContext.getContentFamily;
 
 @Resource(name = RestConstants.VERSION_1 + "/attachment", supportedClass = Attachment.class, supportedOpenmrsVersions = {
         "1.10.*", "1.11.*", "1.12.*" })
-public class AttachmentResource1_10 extends DataDelegatingCrudResource<Attachment> implements Uploadable {
+public class AttachmentResource1_10 extends DataDelegatingCrudResource<Attachment> implements Uploadable, AttachmentService {
```

Comment:
No we **do not** want the REST resource to _implement_ the service layer API, just to use it by getting via `Context`:
```java
AttachmentsService as = Context.getService(AttachmentsService.class);
// use 'as' hereunder...
```
Both `AttachmentsService` and `AttachmentsServiceImpl` have to live in the **/api** subproject.

**NOTE:** Please refactor the API to `AttachmentsService` (plural).